src/atoms/Icon/constants.js

```diff
@@ -129,6 +129,7 @@ module.exports = {
     'healthgenius',
     'healthgenius',
     'heart',
+    'heldHammer',
```

Comment:
This is the main change, adding this icon (which is also included in the description).